app/src/main/java/com/example/easysurvey/MainActivity.kt

```diff
@@ -23,15 +23,15 @@ class MainActivity : ComponentActivity() {
             EasySurveyTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colors.background) {
-                    HomepageButtons()
+                    homepageButtons()
                 }
             }
         }
     }
 }
 
 @Composable
-fun HomepageButtons() {
+fun homepageButtons() {
```

Comment:
changed function name to lowercase camelcase per the suggestions here: https://kotlinlang.org/docs/coding-conventions.html#source-file-organization

Reply:
I really want to agree with you on this, as it is normally the case, but composable functions should be nouns, and PascalCase, believe it or not.  
https://developer.android.com/kotlin/style-guide